src/test/java/com/github/tomakehurst/wiremock/matching/MockRequest.java

```diff
@@ -246,20 +246,9 @@ public boolean isMultipart() {
 
   @Override
   public Part getPart(final String name) {
-    return (getParts() != null && name != null)
-        ? from(multiparts)
-            .firstMatch(
-                new Predicate<Part>() {
-                  @Override
-                  public boolean apply(Part input) {
-                    if (name.equals(input.getName())) {
-                      return true;
-                    }
-                    return false;
-                  }
-                })
-            .get()
-        : null;
+    return getParts() == null || name == null
+        ? null
+        : from(multiparts).firstMatch(input -> name.equals(input.getName())).get();
```

Comment:
I'd personally prefer we get rid of the ternary expression and replace it with an `if ... else`

Reply:
Better indeed; changed.